en/docs/tryout-flows/accelerator-with-is-and-apim/try-out-payments-flow-dcr.md

```
@@ -362,3 +379,22 @@ The TPP must ensure that the Initiation and Risk sections of payment match the c
     }
 }
 ```
+
+#### With UK JWS Header Processing Policy
+
+Just like Step 2, if you have enabled the [UK JWS Header Processing Policy](../../learn/uk-jws-header-processing-policy.md), this request **must be signed** with a Detached JWS using your TPP Private Key.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)